src/SSVOpenHexagon/Components/CPlayer.cpp

```diff
@@ -188,9 +232,9 @@ void CPlayer::update(HexagonGame& mHexagonGame, ssvu::FT mFT)
 
     const sf::Vector2f tempPos{ssvs::getOrbitRad(startPos, angle, radius)};
     const sf::Vector2f pLeftCheck{
-        ssvs::getOrbitRad(tempPos, angle - ssvu::piHalf, 0.01f)};
+        ssvs::getOrbitRad(tempPos, angle - ssvu::piHalf, 1.01f)};
     const sf::Vector2f pRightCheck{
-        ssvs::getOrbitRad(tempPos, angle + ssvu::piHalf, 0.01f)};
+        ssvs::getOrbitRad(tempPos, angle + ssvu::piHalf, 1.01f)};
```

Comment:
Why was this changed?

Reply:
idk, was testing stuff and didn't notice any difference so I guess I left it for more precise collision detection...? Eh, idk honestly but this part will need to be changed actually dues to the custom walls being a complete whacks to properly calculate for collisions, so we need to leave it as just:

```cpp
if(wall.isOverlapping(pos)){
    kill(...);
}
```